### ISSUE TITLE
Downloads are disabled.

### ISSUE THREAD

**Issue by bennodxb** [piejanssens/premiumizer#214]
Hi. Everything was going well up until I updated just now. Now it says "downloads are disabled." They are enabled in settings.
 I use the built in downloader mainly because it worked well and aria was too hard for me to set up.
I restarted the app. That didn't work. 
thanks

**Comment by neox387** [comment]
Should say something in the log even without debug

**Comment by bennodxb** [comment]
Thanks, I looked at the log now. It says because download location is empty, which it is, Would the update remove that? I can't remember ever putting anything in there, I thought the download locations were set in the categories section.

**Comment by neox387** [comment]
It's the other way around xD

You set for example as Download location c:\Premiumizer\downloads

And the category tv will be put in
c:\Premiumizer\downloads\tv then

**Comment by neox387** [comment]
so I guess everything works after u put in the download location folder ?

**Comment by bennodxb** [comment]
Just woke up, I'll try it thanks.

**Comment by gh0stwolf** [comment]
Any luck? I am having the exact same error. I can upload just fine just downloading appears to be an issue. It says "downloads are disabled". I tried changing folders in the cloud to the category in the app to no avail..

**Comment by neox387** [comment]
First use latest version.

Changing maps on the cloud is not gonna do anything but break things possibly.

Premiumizer does not put things in maps on the cloud just locally

**Comment by neox387** [comment]
downloads are disabled it should display in the log why this is if you enabled downloads.

You must put in a download location for example. 

Then on the main screen you can select a category and the download will start.

But with changing stuff database might be broken so you could delete the database files and start premiumizer again

**Comment by gh0stwolf** [comment]
I am using the docker version from piejanssens/premiumizer:latest.

I think I just found out what was going on. I did like you said and put the files in the root directory in the cloud and deleted the db file. The app reloaded everything and I'm no longer getting a download disabled issue. It said waiting for category but wasn't letting me change categories. I was using opera/brave browser. Switched over to IE and I was able to change categories all is good. Have to figure out which setting in Opera is blocking changing the categories button. Otherwise I am up and running!!
